cli/running/writer.go

```diff
@@ -101,7 +101,12 @@ func newColorizedWriter(prefix string) *ColorizedWriter {
 		out: os.Stdout,
 	}
 
-	prefixColor := nexPrefixColor()
-	writer.prefix = prefixColor.Sprintf("%s | ", prefix)
+	if prefix != "" {
```

Comment:
Maybe you need just "newDummy"/"newNoPrefix" writer to create it at https://github.com/tarantool/cartridge-cli/pull/463/files#diff-44e2d99ca4c277c2ae3e6bbd3ea8515486d311ab90fb3578ffd8e194bffa77fdR190